libs/admin/src/lib/components/user-edit/user-edit.component.html

```diff
@@ -1,101 +1,105 @@
-                <mat-option
-                  *ngFor="let workgroup of filteredWorkgroups"
-                  [value]="workgroup.name"
-                  (onSelectionChange)="addWorkgroupRole($event, workgroup.id)"
-                  [disabled]="isUserPartOfWorkgroup(workgroup.id)"
-                >
-                  {{ workgroup.name }}
-                </mat-option>
-              </mat-autocomplete>
-            </mat-form-field>
-          </form>
-        </div>
-        <div>
-          <table mat-table [dataSource]="userWorkgroups$" class="mat-elevation-z1">
-            <!-- ID Column -->
-            <ng-container matColumnDef="name">
-              <th mat-header-cell *matHeaderCellDef>Name</th>
-              <td mat-cell *matCellDef="let workgroup">{{ workgroup.name }}</td>
-            </ng-container>
+<asset-sg-detail-section title="{{ 'admin.userPage.general' | translate }}">
+  <div class="user-data">
+    @if (user) {
+    <div class="filter-container">
+      <asset-sg-filter-selector
+        title="{{ 'admin.userPage.lang' | translate }}"
+        [initialValues]="[user.lang]"
+        [values]="languageSelector"
+        [multiple]="false"
+        (filterChanged)="languageChanged($event)"
+      ></asset-sg-filter-selector>
+    </div>
 
-            <!-- Role Column -->
-            <ng-container matColumnDef="role">
-              <th mat-header-cell *matHeaderCellDef translate>admin.role</th>
-              <td mat-cell *matCellDef="let workgroup">
-                <mat-form-field appearance="fill" subscriptSizing="dynamic" class="workgroups__table__form-field">
-                  <mat-label translate>admin.role</mat-label>
-                  <mat-select [value]="workgroup.role" (selectionChange)="updateWorkgroupRole($event, workgroup.id)">
-                    <mat-option *ngFor="let role of roles" [value]="role">
-                      {{ role }}
-                    </mat-option>
-                  </mat-select>
-                </mat-form-field>
-              </td>
-            </ng-container>
+    <mat-checkbox
+      [checked]="user.isAdmin"
+      (change)="isAdminChanged($event)"
+      color="primary"
+      class="checkbox"
+      [disabled]="isCurrentUser"
+    >
+      <span translate>admin.userPage.admin</span>
+    </mat-checkbox>
+    }
+  </div>
+  <div class="workgroups">
+    <div class="workgroups__header"></div>
+    <div></div>
+  </div>
+</asset-sg-detail-section>
+<asset-sg-detail-section
+  title="{{ 'admin.workgroups' | translate }}"
+  [hasButton]="true"
+  buttonTitle="{{ 'admin.userPage.addWorkgroups' | translate }}"
+  (buttonClick)="openAddWorkgroupToUserDialog()"
+>
+  <asset-sg-search-and-filter
+    (searchTermChanged)="setSearchTerm($event)"
+    (shouldShowFiltersChanged)="toggleFilters($event)"
+  ></asset-sg-search-and-filter>
+  @if (shouldShowFilters) {
+  <div class="filters">
+    <asset-sg-filter-selector
+      title="{{ 'admin.role' | translate }}"
```

Comment:
This reads better, IMO.
```suggestion
      [title]="'admin.role' | translate"
```